static/src/javascripts/projects/common/modules/identity/account-profile.js

```diff
@@ -34,11 +39,14 @@ define([
 
         self.messages = {
             noCorsError: 'Cross-origin resource sharing is not supported by this browser. Please upgrade your browser to use this feature.',
-            noServerError: 'The image upload server could not be reached.'
+            noServerError: 'Oops, the Avatar upload service is currently unavailable. Please try again shortly.',
+            avatarUploadSuccess: 'Thank you for uploading your avatar. It will be checked by Guardian moderators shortly.',
+            avatarUploadFailure: 'Oops, something went wrong. Please try again.'
         };
 
         self.urls = {
-            avatarTokenUrl: 'https://gu-image-upload.appspot.com/upload-endpoint-generator'
+            avatarTokenUrl: 'https://gu-image-upload.appspot.com/upload-endpoint-generator',
```

Comment:
Ah yes - forgot I did that! The avatarTokenUrl stuff is going to disappear once we are happy the new system is working okay, so am inclined to leave this as is for now as it shouldn't exist in a few weeks!